src/api/sendgrid.ts

```diff
@@ -0,0 +1,53 @@
+import mail, { MailDataRequired } from '@sendgrid/mail'
+import apiEnv from './apiEnv';
+
+export interface Personalizations {
+    to: [{
+        email: string,
+        name: string
+    }];
+    dynamic_template_data?: Record<string, string>;
+}
+
+mail.setApiKey(apiEnv.SENDGRID_API_KEY);
+
+const sendEmail = async (personalizations: Personalizations[], template_id: string) => {
+
+    console.log("SendGrid Mail Executed");
+
+    const message: MailDataRequired = {
+        personalizations: personalizations,
+        from: 'yizhang1728@gmail.com', // yuanjian@163
+        templateId: template_id,
+    };
+
+    try {
+        const res = await mail.send(message);
+        console.log("SendGrid Mail sent", res);
+    } catch (e) {
+        console.log(e);
+    }
+}
+
+export const sendEmailToAdmin = async (new_user_email: string) => {
+    const template_id = 'd-2e3aabdb9a3c45a393877596534c57c2'; // temporary holder, change to required template id
+
+    const personalizations: Personalizations[] = [{
+        // change/add codes below to designated email addresses and names
+        to:
+            [
```

Comment:
nit: compress the formatt a bit, can put `to: [{` all onto the same line. same for closing brakets.